### ISSUE TITLE
[BUG] Add accessibility id for Intro video 

### ISSUE THREAD

**Issue by Priyaravi8** [rokwire/illinois-app#1660]
**Describe the bug**
Accessibility id is missing for the intro video

**To Reproduce**
1. Install the new Illinois app
2. Proceed to the Intro video 

**Actual Result**
Accessibility id is not provided to the video

**Expected behavior**
Add accessibility id to the video to validate in the automation script

**Smartphone (please complete the following information):**
 - Device: [e.g. iPhone and Android]
 - Version [e.g. 3.3.22]

**Comment by vburgett** [comment]
Do we plan on updating the intro video for 4.0? @johnmpaul @joegrohens

**Comment by TodorBachvarov** [comment]
Added Semantics label "Onboarding video". This should provide the semantics id. Added in version 4.1.7
